components/mpas-ocean/cime_config/buildnml

```diff
@@ -1029,6 +1029,8 @@ def buildnml(case, caseroot, compname):
             lines.append('    <var name="normalGMBolusVelocitySquared"/>')
             lines.append('    <var name="gmKappaScaling"/>')
             lines.append('    <var name="gmBolusKappa"/>')
+            lines.append('    <var name="mocStreamvalLatAndDepthGM"/>')
+            lines.append('    <var name="mocStreamvalLatAndDepthRegionGM"/>')
```

Comment:
@vanroekel I put these in the monthly time average, with the other GM variables.

Reply:
great, thanks!